pkg/local_object_storage/blobstor/fstree/fstree.go

```diff
@@ -71,27 +71,27 @@ func stringifyAddress(addr oid.Address) string {
 	return addr.Object().EncodeToString() + "." + addr.Container().EncodeToString()
 }
 
-func addressFromString(s string) (*oid.Address, error) {
-	ss := strings.SplitN(s, ".", 2)
-	if len(ss) != 2 {
-		return nil, errors.New("invalid address")
+func addressFromString(s string) (oid.Address, error) {
+	i := strings.IndexByte(s, '.')
```

Comment:
some bench to commit/PR decs about enhancement?

Reply:
Fixed